### ISSUE TITLE
[Feature Request] Add standardization/normalization information into torchvision.datasets

### ISSUE THREAD

**Issue by frgfm** [pytorch/vision#593]
Hi,

I was trying to get myself a very clean training pipeline using torchvision datasets and PyTorch dataloader. In order to test this quickly, I've been working on MNIST.

So, I downloaded the dataset using the usual command specifying a transform:
`train_set = torchvision.datasets.MNIST(root='./data', train=True, download=True, transform=transform)`

I found on GitHub the mean and std of the set and used then for the transform:
`transform = transforms.Compose([transforms.ToTensor(), transforms.Normalize((0.1307,), (0.3081,))])`

My understanding is that the different datasets are added (picking the source for download) by the torchvision team. Would it be possible then to include at least the mean and std for the different datasets ? (as an attribute of torchvision.datasets.MNIST for instance or perhaps directly passing an option normalize=True)

I understand it would require some computation but maybe requesting it for each dataset source could be an option ?

**Comment by frgfm** [comment]
I forgot to answer back, thanks for your reply @fmassa ! Apologies

I have been taking similar approach to your implementation suggestion so far, I was looking for further support by the the library but I now understand better why it wouldnt be possible for torchvision to include it directly

Consistency & user expectations are indeed big counterarguments, thanks again!
